app/src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -418,16 +422,31 @@ public boolean onPreferenceClick(Preference preference) {
                             .setNegativeButton(android.R.string.no, null)
                             .show();
                     return true;
-                case KEY_STRESET:
-                    final Intent intent = new Intent(getActivity(), SyncthingService.class)
-                            .setAction(SyncthingService.ACTION_RESET);
+                case KEY_STRESETDATABASE:
```

Comment:
You should put an underscore, like `KEY_ST_RESET_DATABASE` (also for the action string below).

Reply:
Ok, commit https://github.com/syncthing/syncthing-android/pull/1061/commits/21fd135c264bcdb0a7ee5cc9d6bba5daca1ef560